driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -724,6 +694,26 @@ public void waitForDown(int node) {
         TestUtils.waitUntilPortIsDown(addressOfNode(node));
     }
 
+    @Override
+    public ProtocolVersion getDesiredProtocolVersion() {
+        VersionNumber version = getCassandraVersion();
```

Comment:
this is only checking cassandra versions, shouldn't it be checking dse versions as well? Anyway as per my previous comment I think a single CCMBridge instance should carry only one version, and a boolean flag to indicate if it is a DSE cluster or not.